gangcaiyoule/myai_backend/joint.py

```
@@ -0,0 +1,91 @@
+    num_rows = math.ceil(len(images_sorted) / max_per_row)
+    total_width = img_width * max_per_row
+    total_height = img_height * num_rows
+    merged_image = Image.new('RGB', (total_width, total_height), color=(255, 255, 255))
+    for idx, (img, _) in enumerate(images_sorted):
+        row = idx // max_per_row
+        col = idx % max_per_row
+        x = col * img_width
+        y = row * img_height
+        merged_image.paste(img, (x, y))
+    merged_image.save(save_path)
+    print(f"已按色相排序并拼接，共 {len(images_sorted)} 张图，保存为 {save_path}")
```

Ruff (0.13.3)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)